tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "removeComments": true,
     "noUnusedLocals": true,
     "noImplicitAny": true,
-    "lib": ["es2015", "dom"],
+    "lib": ["dom", "es2022"],
```

Comment:
Do we need `es2022` or can we do with a less restrictive library? Would be good to keep minimal requirements where possible.

Reply:
Turns out we are not using much from es2022 library. Reverted existing ones to be compatible with es2015.